packages/profile-sync-controller/src/shared/storage-schema.ts

```diff
@@ -9,31 +9,33 @@ import { createSHA256Hash } from './encryption';
  */
 const ALLOW_ARBITRARY_KEYS = 'ALLOW_ARBITRARY_KEYS' as const;
 
+export enum UserStorageFeatureNames {
+  Notifications = 'notifications',
+  Accounts = 'accounts_v2',
+  Networks = 'networks',
```

Comment:
Just want to make 100% sure these names are identical to what they were before?

Reply:
Also I see that account_v2 change lol.

We might not need this as we are resolving the issue through a delete endpoint.

But not against it if we want a fresh clean slate

Reply:
Confirmed that they are identical! :) 
I'm leaning on the side of starting from scratch after having so many devs impacted, let's keep `accounts_v2` AND add a safeguard deleting possible bogus entries. I'll have a PR ready probably tomorrow about this.